src/Common/src/System/Net/CookieParser.cs

```diff
@@ -667,6 +667,17 @@ internal Cookie Get()
                                         {
                                             cookie.Expires = expires;
                                         }
+                                        else if (DateTime.TryParseExact(CheckQuoted(_tokenizer.Value),
+                                            "d MMM yyyy HH:mm:ss UTC",
```

Comment:
Is UTC required to be capitalized? What if it is lower case? Will the browsers recognize it as well?

Reply:
We should use the same parsing rules as-if the "UTC" was "GMT".  So, if the RFC allows for case insensitivity for GMT, we should do the same.

Reply:
Also, we should consider using this class which is already in use for the rest of the HTTP stack.  And this has 'UTC' timezone in it as well as part of recent changes.

https://github.com/dotnet/corefx/blob/d7f120a01ac6ea09a2139c99e43d7811950d2f73/src/Common/src/System/Net/HttpDateParser.cs#L5

Reply:
Updated to use HttpDateParser now.